migrations/8_setup_meta_colony.js

```diff
@@ -62,23 +62,45 @@ module.exports = async function (deployer, network, accounts) {
   const ColonyTask = artifacts.require("./ColonyTask");
   const ColonyPayment = artifacts.require("./ColonyPayment");
   const ContractRecovery = artifacts.require("./ContractRecovery");
+  const ColonyArbitraryTransaction = artifacts.require("./ColonyArbitraryTransaction");
 
   const colony = await Colony.new();
   const colonyFunding = await ColonyFunding.new();
   const colonyExpenditure = await ColonyExpenditure.new();
   const colonyRoles = await ColonyRoles.new();
   const colonyTask = await ColonyTask.new();
   const colonyPayment = await ColonyPayment.new();
-  const contractRecovery = await ContractRecovery.deployed();
+  const contractRecovery = await ContractRecovery.new();
```

Comment:
Why this change?

Reply:
This got changed to `ColonyRecovery.new()` originally with my first crack, but when you pointed out we didn't need separate recovery contracts with the implementation, I changed all the `ColonyRecovery`s back to `ContractRecovery`, but obviously missed the `deployed/new` subtlety. Will be changed back.